src/extension.ts

```diff
@@ -47,17 +47,22 @@ export async function activate(context: ExtensionContext) {
     context.subscriptions.push(commands.registerCommand(TfvcCommandNames.Status, () => _extensionManager.Tfvc.TfvcStatus()));
     context.subscriptions.push(commands.registerCommand(TfvcCommandNames.Undo, (...args) => {
             if (args) {
-                //TODO: Multi-select?
                 _extensionManager.Tfvc.TfvcUndo(args[0]);
             } else {
                 _extensionManager.Tfvc.TfvcUndo();
             }
-        }
-    ));
-    context.subscriptions.push(commands.registerCommand(TfvcCommandNames.Exclude, () => _extensionManager.Tfvc.TfvcExclude()));
-    context.subscriptions.push(commands.registerCommand(TfvcCommandNames.ExcludeAll, () => _extensionManager.Tfvc.TfvcExcludeAll()));
-    context.subscriptions.push(commands.registerCommand(TfvcCommandNames.Include, () => _extensionManager.Tfvc.TfvcInclude()));
-    context.subscriptions.push(commands.registerCommand(TfvcCommandNames.IncludeAll, () => _extensionManager.Tfvc.TfvcIncludeAll()));
+        }));
+    context.subscriptions.push(commands.registerCommand(TfvcCommandNames.Exclude, (...args) => {
+            if (args) {
```

Comment:
Since most commands will need this logic, it would be nice to somehow push this into a helper method. Not sure if that is easy or not.